README.md

```diff
@@ -116,10 +117,9 @@ For more info:  [hacktoberfest.com](https://hacktoberfest.com/)
   </tbody>
 </table>
 
-<!-- markdownlint-restore -->
-<!-- prettier-ignore-end -->
-
-<!-- ALL-CONTRIBUTORS-LIST:END -->
+<!-- List of contributors with their GitHub profiles and contributions -->
+<!-- Include avatars, links, and contribution categories -->
+<!-- Update this section as more contributors join -->
```

Comment:
The original lines should not be changed here, because they are for the all-contributors bot.